src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -100,6 +119,33 @@ public PersonCard(Person person, int displayedIndex) {
             preferred.getStyleClass().add("button1");
         }
 
+        cir2.setStroke(Color.AQUAMARINE);
+        Image im = new Image(social.getImageUrl(), false);
+        cir2.setFill(new ImagePattern(im));
+        cir2.setEffect(new DropShadow(+25d, 0d, +2d, Color.AQUAMARINE));
```

Comment:
Perhaps it might be better to do styling in the fxml file? Or would it also be possible to abstract these to a stylePersonCard() method.